docs/discovery-search.html

```diff
@@ -572,8 +572,23 @@ <h3 id="filter-title">Filter Results</h3>
                   <label for="language-filter-4">Language</label>
               </li>
               <li>
-                <input id="language-filter-5" type="checkbox" name="language" value="Language" />
-                <label for="language-filter-5">Language</label>
+                <input id="language-filter-6" type="checkbox" name="language" value="Language" />
```

Comment:
Did you intentionally skip `language-filter-5`?

Reply:
Okay, doesn't look like it matters though.

Reply:
added it back